deploy/kubernetes/manifests/controller.yaml

```diff
@@ -18,28 +18,16 @@ spec:
     spec:
       serviceAccount: gcp-filestore-csi-controller-sa
       containers:
-        - name: csi-external-attacher
```

Comment:
Missing the CSIDriver manifest to specify that attach is not required

Reply:
I have added the CSIDriver in the driver_start script. 
https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/blob/master/deploy/kubernetes/driver_start.sh#L11

Reply:
@mattcary  reply to your question below